worker_test.go

```diff
@@ -217,6 +217,41 @@ func TestRequestFromJS(t *testing.T) {
 	}
 }
 
+func TestModules(t *testing.T) {
+	worker := New(func(msg []byte) []byte {
+		t.Fatal("shouldn't recieve Message")
+		return nil
+	})
+	exportCode := `
```

Comment:
s/exportCode/dependencyCode
Or better yet, don't name the string and just put it into the function call below

Reply:
I believe this was attempting to shorten the lines.  I can inline it.